### ISSUE TITLE
Remove references to companies to be vendor-neutral

### ISSUE THREAD

**Issue by rzo1** [apache/incubator-stormcrawler#1170]
Carefully check for links to vendor / company for pre-ASF era. As an ASF project, we need to be vendor neutral, i.e. we need to remove references to the commercial support company.

Maybe we can do something similar to https://tomee.apache.org/commercial-support.html

**Comment by rzo1** [comment]
This also includes pages on the Wiki, i.e. links to sponsoring an entity (which do not work anyway)

**Comment by jnioche** [comment]
The sponsoring has been removed from the WIKI
What about adding a Commercial Support section on the WIKI and remove the ref to DiPe on the README?

**Comment by rzo1** [comment]
Yes. We could phrase it like it is done on TomEE: https://tomee.apache.org/commercial-support.html wdyt?

**Comment by jnioche** [comment]
the README bow points to the support page of the site. I can't see any references to companies apart from external repositories that could be useful,.
@rzo1 can you double check and close the issue if appropriate? thanks!
